src/main/java/goblin/compendium_entry/CompendiumEntry.java

```diff
@@ -0,0 +1,30 @@
+package goblin.compendium_entry;
+
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.util.UUID;
+
+@ToString
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+@Getter
+@Setter
+public class CompendiumEntry {
+
+    @EqualsAndHashCode.Include
+    private final UUID id;
+    private String name;
+    private EntryType entryType;
+    private Boolean isSoftDeleted;
```

Comment:
You will also need date where the soft delete will occur, now it is impossible to know when entry should be deleted.